library/config.py

```diff
@@ -76,6 +76,8 @@ def compute(self) -> dict:
             # it will not replace what's in yaml
             version = self.version if self.version else self.version_today
             config = self.parsed_rendered_template(version=version)
+            # Overwrite even if version specified
```

Comment:
`# Overwrite with version passed through CLI, even if version is specified in the config file`